### DISCUSSION TITLE
Models on example photos

### DISCUSSION THREAD

**Discussion by davidrios** [advanced-cropper/vue-advanced-cropper#193] [Ideas]
Hey man, thanks for the library, awesome work. Awesome documentation also.

Speaking of documentation, I think it's kind of weird that pretty much all example photos are of pretty/sexy women, and that could put some people off. I like pretty women just as much, but it may be better to use more neutral photos, in the interest of better inclusivity, and possibly better adoption of the library.

I'm totally up to do it if you agree.

Cheers

**Comment by add-n2x** [comment]
@davidrios thanks for bringing this up.

> I think it's kind of weird that pretty much all example photos are of pretty/sexy women

...and a soldier. :sob: 

> it may be better to use more neutral photos, in the interest of better inclusivity

Yep. 

The quote "If you leave a white post alone it will soon be a black post." on the authors profile is by G.K. Chesterton. I have the slight feeling, that this is not the library to use, if you are standing for non-conservative and inclusive world views.

**Reply by minasmask** [reply]
Given these points, I encourage everyone to explore alternatives. There are numerous libraries and resources that emphasize diversity and inclusivity. By choosing such alternatives, we can ensure that our work and the tools we use align with our commitment to fostering a more equitable world.
